storage/Devices/PartitionableImpl.cc

```diff
@@ -118,7 +119,23 @@ namespace storage
 	    if (get_region().get_length() != parted.get_region().get_length())
 		ST_THROW(Exception("different size reported by kernel and parted"));
 
-	    PartitionTable* pt = create_partition_table(parted.get_label());
+	    PtType label = parted.get_label();
+
+	    // parted reports DASD partition type for implicit partition
```

Comment:
NP: leftover partition word in "DASD partition type"?

Reply:
No, type should be table.